.circleci/config.yml

```diff
@@ -587,6 +610,7 @@ workflows:
       - assemble-purchase-tester
       - run-backend-integration-tests
       - verify-revenuecatui-snapshots
+      - run-revenuecatui-ui-tests
```

Comment:
I was wondering about whether to run these tests on all commits or just on release branches... I think we can keep it here for now, as we work on paywalls, but maybe move it to release branches in the future. Lmk if you think otherwise!